spec/features/attestations_spec.rb

```diff
@@ -0,0 +1,218 @@
+        within('.authenticated-user-sidemenu') do
+          expect(page).not_to have_content(I18n.t('.shared.user_signed_in_side_menu.download_attestations'))
+        end
+      end
+    end
+
+    context 'when user has attestations role' do
+      let(:user) { create :user, :with_jwt_specific_roles, specific_roles: ['attestations_fiscales'] }
+
+      it 'shows menu item for attestations download' do
+        within('.authenticated-user-sidemenu') do
+          expect(page).to have_content(I18n.t('.shared.user_signed_in_side_menu.download_attestations'))
+        end
+      end
+    end
+  end
+
+  describe 'select menu' do
+    subject(:visit_attestations) { visit profile_attestations_path }
+
+    before do
+      login_as(user)
+      visit_attestations
+      FactoryBot.rewind_sequences
+    end
+
+    context 'when user has tokens with attestation roles' do
+      let(:user) { create :user, :with_jwt_specific_roles, specific_roles: ['attestations_fiscales'] }
+
+      it 'has a select list' do
+        expect(page).to have_select('token')
+      end
+
+      it 'select list has 3 options' do
+        expect(page.all('select#token option').map(&:text)).to eq([
+          '',
+          'JWT with roles: ["attestations_fiscales"]',
+          'JWT with no roles'
+        ])
+      end
+
+      it 'tell user to select a token' do
+        expect(page).to have_content('Selectionnez un token pour voir les droits')
+      end
+
+      context 'when user select token with roles', js: true do
+        before { select('JWT with roles: ["attestations_fiscales"]', from: 'token') }
+
+        it 'shows roles' do
+          expect(page).to have_content("Attestations disponibles pour ce token :\nAttestations fiscales")
+        end
+      end
+
+      context 'when user select token with no roles', js: true do
+        before { select('JWT with no roles', from: 'token') }
+
+        it 'shows no roles' do
+          expect(page).to have_content('Aucune attestation disponible avec ce token')
+        end
+      end
+
+      context 'when user comes back to empty selection', js: true do
+        before do
+          select('JWT with roles: ["attestations_fiscales"]', from: 'token')
+          select('', from: 'token')
+        end
+
+        it 'ask to select a token' do
+          expect(page).to have_content('Veuillez sélectionner un token dans la liste.')
+        end
+      end
+    end
+
+    context 'when user has no token with attestation roles' do
+      let(:user) { create(:user) }
+
+      it 'does not have a select list' do
+        expect(page).not_to have_select('token')
+      end
+    end
+  end
+
+  describe 'search', js: true do
+    subject(:search) do
+      login_as(user)
+      visit profile_attestations_path
+      select(token, from: 'token')
+      fill_in('search_siret', with: siret)
+      click_button('search')
+    end
+
+    let(:user) { create :user, :with_jwt_specific_roles, specific_roles: ['attestations_fiscales'] }
```

Comment:
```suggestion
    let(:user) { create(:user, :with_jwt_specific_roles, specific_roles: ['attestations_fiscales']) }
```

consistance

très étrange que ça passe rubocop ça